File: twiml/voice/transcription/transcription-intelligenceservice/transcription-intelligenceservice.js
const VoiceResponse = require('twilio').twiml.VoiceResponse;

const response = new VoiceResponse();
const start = response.start();
start.transcription({statusCallbackUrl: 'https://example.com/your-callback-url', intelligenceService: 'GAaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa'});

console.log(response.toString());

File: twiml/voice/transcription/transcription-intelligenceservice/transcription-intelligenceservice.cs
using System;
using Twilio.TwiML;
using Twilio.TwiML.Voice;


class Example
{
    static void Main()
    {
        var response = new VoiceResponse();
        var start = new Start();
        start.Transcription(statusCallbackUrl: "https://example.com/your-callback-url", intelligenceService: "GAaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa");
        response.Append(start);

        Console.WriteLine(response.ToString());
    }
}
